.circleci/config.yml

```diff
@@ -67,11 +67,25 @@ commands:
             - /home/circleci/.composer/cache/
       - store_artifacts:
           path: ./artifacts
-      - slack/status:
-          fail_only: true
-          failure_message: ':red_circle: M2: A $CIRCLE_JOB job has failed!'
-          only_for_branches: master
-          webhook: $SLACK_GREENKEEPER_WEBHOOK
+      - run:
+          name: "Setup branch string"
+          when: on_fail
+          command: |
+            if [[ ${CIRCLE_BRANCH:0:3} == ci/ ]] && grep -n "^${CIRCLE_BRANCH:3}$" ./auto-rebase-branchess.txt >> /dev/null ; then
+              export NOTIFY_FLAG=true
+            else
+              export NOTIFY_FLAG=false
+            fi
+            if [[ ${CIRCLE_BRANCH} == "master" ]]
```

Comment:
Minor style thing but nested if will make sure we are  not doing an unnecessary check 

```
if [[ ${CIRCLE_BRANCH} == "master" ]]; then
	echo true
else
	if [[ ${CIRCLE_BRANCH:0:3} == ci/ ]] && grep -n "^${CIRCLE_BRANCH:3}$" ./auto-rebase-branches.txt >> /dev/null ; then
	  echo true
	else
	  echo false
	fi
fi
```